cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -483,13 +483,7 @@ int32_t cros_gralloc_driver::get_backing_store(buffer_handle_t handle, uint64_t
 		return -EINVAL;
 	}
 
-	auto buffer = get_buffer(hnd);
```

Comment:
Could we have one flag to tell this is from Android specific changes? For example:
#ifdef USE_GRALLOC1

#endif

Reply:
Done, Please review again.

Reply:
LGTM, thanks Dong!